### PR TITLE
Install and configure automation agents

### PR DESCRIPTION
**Pull request by MihaiBojin**

### CODE REVIEW

File: mongodb/util/helpers.go [hunk 1]
```diff
@@ -25,3 +35,39 @@ func LogNonNilError(err error) {
...
+	if len(tentative) < 2 {
+		// if the contents does not contain a dot, attempt to split by underscore
+		tentative = strings.SplitN(part, "_", 2)
+	}
+
+	if len(tentative) < 2 {
+		// could not find 2 parts, set both as the same
+		firstName = tentative[0]
+		lastName = tentative[0]
+		emailAddress = contents
```
Comment (savannaendicott) [inline mongodb/util/helpers.go]: can remove this line - already set on line 53

File: examples/standalone-in-docker/main.tf [hunk 2]
```diff
@@ -18,83 +18,88 @@ resource "docker_container" "mdb0-0" {
...
+# Generate an encryption key for Ops Manager
+resource "random_string" "encryptionkey" {
+  length = 24
+  special = true
+}
+
+resource "random_string" "globalownerpassword" {
+  length = 12
+  min_numeric = 4
+  min_special = 1
```
Comment (savannaendicott) [inline examples/standalone-in-docker/main.tf]: we need to have at least one letter as well, although unlikely we should remove the possibility by adding `min_lower = 1` or `min_upper = 1`

File: mongodb/util/helpers.go (hunk 1 above)
Comment (MihaiBojin) [reply mongodb/util/helpers.go]: good catch - the logic was bad so I fixed it.

File: examples/standalone-in-docker/main.tf (hunk 2 above)
Comment (MihaiBojin) [reply examples/standalone-in-docker/main.tf]: good point, unlikely but not impossible :)

**Review by savannaendicott — APPROVED** [review]
LGTM!

**Pull request merged**